src/components/pathway-progress/PathwayProgressListingPage.jsx

```diff
@@ -39,9 +39,9 @@ const PathwayProgressListingPage = () => {
 
   return (
     <div className="py-5">
-      {pathwayProgressData?.length > 0 ? (
+      {data?.length > 0 ? (
```

Comment:
nit: the `?` here should be unnecessary because of the `if (!data)` conditional above.